model_analyzer/record/metrics_manager.py

```diff
@@ -292,6 +292,11 @@ def _create_model_variants(self, run_config):
 
                 self._create_model_variant(original_name, bls_composing_config)
 
+                bls_original_composing_config = BaseModelConfigGenerator.create_original_config_from_variant(
```

Comment:
Is this needed? Or do you have access to the original anywhere here in this function? It seems like we are creating variants from original and then original from variant, but the function names may just be misleading

Reply:
The originally named config is just the default, which is not what we want. Need to create an originally (unmodified) named model config, that has been modified with whatever the RCG has chosen. I'm open to a better name suggestion for this method.

Reply:
ah, so 'original' in one case is the base config, and 'original' here is the variant config with the base name. Confusing. Yes, one of the two mentions of 'original' should be updated.

Reply:
No, in both cases we are taking in the variant config.